File: 조현창/11098 첼시를 도와줘.cpp
#include<iostream>
#include<string>
using namespace std;

int main(){
  int n;
  cin >> n;
  while(n--){
    int p, max = 0;
    string answer;
    cin >> p;
    while(p--){
      int input;
      string name;
      cin >> input >> name;
      if(input > max){
        answer = name;
        max = input;
      }
        
    }
    cout << answer << '\n';
  }
}

File: 1학기 스터디/김예원/11098 첼시를 도와줘.py
n = int(input())
for i in range(n):
    p = int(input())
    max = 0
    whoAreMax = ""
    for j in range(p):
        price, name = input().split()
        if int(price) > max:
            max = int(price)
            whoAreMax = name
            
    print(whoAreMax)